lib/GenericsAPI/Utils/BIOMUtil.py

```diff
@@ -35,10 +35,12 @@ def _file_to_data(self, file_path, refs, matrix_name, workspace_id):
             name = matrix_name + "_row_attributes"
             data['row_attributemapping_ref'] = self._metadata_to_attribute_mapping(
                 table._observation_ids, table._observation_metadata, name, workspace_id)
+            data['row_mapping'] = {x: x for x in matrix_data['row_ids']}
```

Comment:
without checking unmapped_ids, I think default should be attribute instances keys.